content/400-reference/200-api-reference/050-prisma-client-reference.mdx

```diff
@@ -1005,6 +1005,111 @@ When your application tries to perform two or more concurrent upsert operations,
 
 Handle the P2002 error in your application code. When it occurs, retry the upsert operation to update the row.
 
+#### Database upserts
+
+Where possible, Prisma Client hands over an `upsert` query to the database, instead of handling the upsert itself. This is called a _database upsert_.
+
+Database upserts have the following advantages:
+
+- They are faster than regular upserts
```

Comment:
"regular upserts" might be misleading here, the database ones are "regular" for most.
"Prisma level upserts" maybe?

Reply:
Good point - I'll revise

Reply:
Done